st2client/st2client/client.py

```diff
@@ -162,6 +164,23 @@ def __init__(self, base_url=None, auth_url=None, api_url=None, stream_url=None,
         self.managers['UserRoleAssignment'] = ResourceManager(
             models.UserRoleAssignment, self.endpoints['api'], cacert=self.cacert, debug=self.debug)
 
+    @add_auth_token_to_kwargs_from_env
+    def get_user_info(self, **kwargs):
+        """
+        Retrieve information about the current user which is authenticated against StackStorm API.
+
+        :rtype: ``dict``
+        """
+        url = '/user'
+        client = httpclient.HTTPClient(root=self.endpoints['api'], cacert=self.cacert,
+                                       debug=self.debug)
+        response = client.get(url=url, **kwargs)
+
+        if response.status_code != 200:
```

Comment:
Use the constant as opposed to 200.

Reply:
Will do.

Reply:
Btw, I just follow the existing style (other code doesn't use constants), but fine.